src/components/organizations/controllers.test.tsx

```diff
@@ -289,3 +297,368 @@ describe(updateOrgData, () => {
+
+    const token = new Token(accessToken, [tokenKey]);
+    const ctx: IContext = createTestContext({ token });
+
+    it('should return an error', async () => {
+      await expect(emailManagersFormPost(ctx, {}, {} as any)).rejects.toThrow(
+        /Not a platform admin/,
+      );
+    });
+  });
+  describe('when platform admin', () => {
+    let ctx: IContext;
+    let nockCF: nock.Scope;
+    let nockZD: nock.Scope;
+    let nockAccounts: nock.Scope;
+    let nockUAA: nock.Scope;
+
+    const organizationGUID = 'a7aff246-5f5b-4cf8-87d8-f316053e4a20';
+    const spaceGUID = '5489e195-c42b-4e61-bf30-323c331ecc01';
+    const mockAccountsUser = {
+      user_email: 'tester@test.com',
+      username: 'tester@test.com',
+      user_uuid: 'uaa-id-253',
+    }
+    const platformAdmin = {
+      id: 'platform-admin',
+      userName: 'platform-admin@fake.cabinet-office.gov.uk',
+      name: { familyName: 'Stub', givenName: 'User' },
+      emails: [{
+        value: 'platform-admin@fake.cabinet-office.gov.uk',
+        primary: false,
+      }],
+    }
+
+    beforeEach(() => {
+      nock.cleanAll();
+
+      const time = Math.floor(Date.now() / 1000);
+      const rawToken = {
+        exp: time + 24 * 60 * 60,
+        origin: 'uaa',
+        scope: [CLOUD_CONTROLLER_ADMIN],
+        user_id: 'platform-admin',
+      };
+      const accessToken = jwt.sign(rawToken, tokenKey);
+
+      const token = new Token(accessToken, [tokenKey]);
+
+      ctx = createTestContext({ token });
+
+      nockZD = nock(ctx.app.zendeskConfig.remoteUri);
+      nockCF = nock(ctx.app.cloudFoundryAPI);
+      nockAccounts = nock(ctx.app.accountsAPI);
+      nockUAA = nock(ctx.app.uaaAPI);
+
+
+      nockCF
+        .get(`/v2/organizations/${organizationGUID}/user_roles`)
+        .times(2)
+        .reply(200, cfData.userRolesForOrg)
+
+        .get(`/v2/organizations/${organizationGUID}`)
+        .reply(200, defaultOrg())
+
+        .get(`/v2/organizations/${organizationGUID}/spaces`)
+        .times(2)
+        .reply(200, cfData.spaces)
+      
+    });
+
+    afterEach(() => {
+      nockAccounts.done();
+      nockUAA.done();
+      nockCF.on('response', () => {
+        nockCF.done();
+      });
+  
+      nock.cleanAll();
+    });
+
+    it('should create a zendesk ticket correctly', async () => {
+      nockZD
+        .post('/tickets.json')
+        .reply(201, {});
+
+      nockAccounts
+        .get('/users/uaa-id-253')
+        .reply(200, mockAccountsUser)
+
+      nockUAA
+        .get('/Users/platform-admin')
+        .reply(200, platformAdmin)
+        .post('/oauth/token?grant_type=client_credentials')
+        .reply(200, '{"access_token": "FAKE_ACCESS_TOKEN"}');
+
+      const response = await emailManagersFormPost(ctx, { organizationGUID: organizationGUID }, {
+        managerType: 'org_manager',
+        message: 'message text',
+        space: spaceGUID,
+      } as any);
+
+      expect(response.status).toBeUndefined();
```

Comment:
How come response status is undefined?

Reply:
technically, we never set a status for these pages, so that like can just be removed I guess. Currently it matches all other tests for the controller

We could manually set 200 on the success page

Reply:
Aha, OK. I understood this as testing the response code from Zendesk. If it's normal, leave it as is.